deploy/crds/network-addons-config-example.cr.yaml

```diff
@@ -5,9 +5,7 @@ metadata:
   name: cluster
```

Comment:
This file is generated. Please edit `pkg/components/components.go` `GetCR()`. Then run `make manifests`

Reply:
done